lib/needlessDisables.js

```diff
@@ -77,8 +89,10 @@ module.exports = function(
 				// If this range is unused and no equivalent is marked,
 				// mark this range as unused
 				if (!range.used && !alreadyMarkedUnused) {
-					range.unusedRule = rule;
-					unused.ranges.push(_.omit(range, 'strictStart', 'strictEnd'));
+					unused.ranges.push({
+						..._.omit(range, 'strictStart', 'strictEnd'),
```

Comment:
Looks like Morse code :) Could you put `_.omit(range, 'strictStart', 'strictEnd')` in a variable so destructuring would look simpler and more understandable, please?

Reply:
done